pio/src/iSpindel.cpp

```diff
@@ -997,8 +997,18 @@ float getTilt()
 
   for (; i < MEDIANROUNDSMAX; i++)
   {
+    uint8_t wait = 0;
     while (!accelgyro.getIntDataReadyStatus())
+    {
       delay(2);
+
+      if (++wait > 50)
+          break;
```

Comment:
The error print can be here before the break and hence the second conditional can be omitted then

Reply:
Yes! Updated, thanks.